marquez_airflow/dag.py

```diff
@@ -115,12 +115,12 @@ def handle_callback(self, *args, **kwargs):
             task_instances = dagrun.get_task_instances()
             for ti in task_instances:
                 extractor = self._extractors.get(ti.operator.__class__)
-                steps_meta = extractor(ti.operator).extract_on_complete()
+                steps_meta = extractor(ti.operator).extract_on_complete(ti)
                 log.info(steps_meta)
 
-                # TODO: update to marquez
+                # FIXME: replace with update api
                 for step in steps_meta:
-                    self.get_marquez_client().create_job(
+                    self._marquez_client.create_job(
```

Comment:
Using `get_marquez_client()` here is safer because a new client is instantiated if `_marquez_client=None`.

Reply:
done.